### ISSUE TITLE
"Browse files" opens in Gwenview over Dolphin if the former is installed

### ISSUE THREAD

**Issue by DrDoctor13** [lutris/lutris#821]
For some reason, Lutris will only open folders with the "browse files" option in Gwenview. I have Dolphin set as my default file manager, but Gwenview still opens. If I uninstall Gwenview, it opens Dolphin just fine.

**Comment by Adam-Cass** [comment]
I can reproduce this on Antergos with KDE but not on Fedora with KDE. What distro and desktop are you using (I assumed KDE because dolphin and Gwenview are defaults there)?

For some reason Gtk.show_uri isn't respecting the default file manager but I don't know why.

Also these clicks lead to the bug as well: Lutris -> Manage Runners -> Open Runners Folder

**Comment by Adam-Cass** [comment]
I can reproduce this on Antergos with KDE but not on Fedora with KDE. What distro and desktop are you using @DrDoctor13 (I assumed KDE because dolphin and Gwenview are defaults there)?

For some reason Gtk.show_uri isn't respecting the default file manager but I don't know why.

Also these clicks lead to the bug as well: Lutris -> Manage Runners -> Open Runners Folder

**Comment by TingPing** [comment]
> For some reason Gtk.show_uri isn't respecting the default file manager but I don't know why.

A safe guess its its not actually the default: `gio mime inode/directory`

**Comment by Adam-Cass** [comment]
@TingPing gio indeed reports that nothing is set as the default but I think that is because it's set in the desktop mime files. I also found that gio told me nothing was set on the fedora distro that did open files in dolphin by default. Thanks for the tip though because this definitely seems like something funky with the desktop and not Lutris.

@DrDoctor13 I managed to solve the issue for my Antergos KDE machine by
First setting Gwenview as the default file manager in system settings and clicking 'Apply.'
Then setting Dolphin as the default file manager in system settings and clicking 'Apply.'
